internal/provider/test_helpers.go

```diff
@@ -0,0 +1,70 @@
+package provider
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/hcp-sdk-go/clients/cloud-resource-manager/stable/2019-12-10/client/project_service"
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov5"
+	"github.com/hashicorp/terraform-plugin-mux/tf5muxserver"
+	"github.com/hashicorp/terraform-provider-hcp/internal/clients"
+	"github.com/hashicorp/terraform-provider-hcp/version"
+)
+
+var TestProtoV5ProviderFactories = map[string]func() (tfprotov5.ProviderServer, error){
+	"hcp": func() (tfprotov5.ProviderServer, error) {
+		providers := []func() tfprotov5.ProviderServer{
+			providerserver.NewProtocol5(NewFrameworkProvider(version.ProviderVersion)()),
+		}
+		return tf5muxserver.NewMuxServer(context.Background(), providers...)
+	},
+}
+
+// testAccPreCheck verifies and sets required provider testing configuration
+//
+// This PreCheck function should be present in every acceptance test. It ensures
+// testing functions that attempt to call HCP APIs are previously configured.
+func TestAccPreCheck(t *testing.T) {
```

Comment:
This should include the old checks for azure/aws credentials (optionally turned on by tests that require the credentials

Reply:
The tests for the data source that got converted don't require aws and azure credentials. I'm thinking that could potentially be a follow up PR.

Reply:
Sounds good